docker-compose.yml

```diff
@@ -15,7 +15,47 @@ services:
     networks:
       - default
 
+  ####################### Database and API Definition #######################
+  db:
+    image: example-db-img
```

Comment:
the image directive is probably not doing much

Reply:
I will trust your wisdom on docker compose file choices, but I'm not confident in my ability to not break things